rust/tools/authorization-logic/src/test/test_queries.rs

```diff
@@ -0,0 +1,77 @@
+#[cfg(test)]
+pub mod test {
+    use crate::souffle::souffle_interface::*;
+    use std::process::Command;
+
+    // This gives the name of an authorization logic program in
```

Comment:
nit: Comments on a struct should probably be doc comments.

Reply:
fixed now